roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -155,7 +155,7 @@
 - name: Cri-o | write registries configs
   template:
     src: registry.conf.j2
-    dest: "/etc/containers/registries.conf.d/10-{{ item.prefix | default(item.location) | regex_replace(':', '_') }}.conf"
+    dest: "/etc/containers/registries.conf.d/10-{{ item.prefix | default(item.location) | regex_replace(':', '_') | regex_replace('/', '_') }}.conf"
```

Comment:
You don't need to add a second regex filter, just match both `:` and `/` in the first

Reply:
I've switched the regex to `regex_replace(':|/', '_')`